### DISCUSSION TITLE
không thể dùng cái extension của bác nữa

### DISCUSSION THREAD

**Discussion by n0kia2k7** [HoangTran0410/useful-script#5] [Q&A] [answered]
khi cài cái script của bác mở lên nó toàn quay là sao thế bác

**Comment by n0kia2k7** [comment]
![Untitled](https://user-images.githubusercontent.com/111240530/216964534-69fa9093-755a-424c-b606-62b3ec1aa1a7.png)

**Reply by HoangTran0410** [reply, marked as answer]
Bác cài bản mới nhất 1.7 thử lại nha
